### PR TITLE
SITES-000: Add user-facing report of file paths

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- Let users look up their file paths

# Needed By (Date)
- 10.8.18

# Criticality
- How critical is this PR on a 1-10 scale? 4/10
- It would be nice to have once we cut over 800 "people" sites and have to help users with broken links to files

# Steps to Test

1. Check out this branch and enable the module
2. Go to `admin/reports/status`
3. See "File Paths | Your public file path is sites/default/files. Your private file path is sites/default/files/private." (or whatever your file paths are)

# Affected Projects or Products
- ACSF

# Associated Issues and/or People
None

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: stanford_sites_helper.install [hunk 1]
```diff
@@ -8,6 +8,29 @@
...
+  switch ($phase) {
+    case 'runtime':
+      $t = get_t();
+      $file_public_path = variable_get('file_public_path', '');
+      $file_private_path = variable_get('file_private_path', '');
+
+      $requirements['stanford_sites_helper'] = array(
+        'title' => $t('File Paths'),
+        'value' => $t('Your public file path is %file_public_path. Your private file path is %file_private_path.', array('%file_public_path' => $file_public_path, '%file_private_path' => $file_private_path)),
+        'severity' => REQUIREMENT_INFO,
```
Comment (jbickar) [inline stanford_sites_helper.install:20]: `REQUIREMENT_OK` or `REQUIREMENT_INFO`?

Comment (sherakama) [reply stanford_sites_helper.install:20]: INFO as you are not really doing a check for anything but rather, are just displaying information even if it is incorrect.

Comment (jbickar) [reply stanford_sites_helper.install:20]: Yeah, I was debating to check whether the path is set but that way be dragons.

File: stanford_sites_helper.install [hunk 2]
```diff
@@ -8,6 +8,29 @@
...
 
+/**
+ * Implements hook_requirements().
+ */
+function stanford_sites_helper_requirements($phase) {
+  $requirements = array();
+
+  switch ($phase) {
+    case 'runtime':
+      $t = get_t();
```
Comment (sherakama) [inline stanford_sites_helper.install]: I don't think you need to do this on runtime checks. Just install time. You should have the `t()` function available at runtime.

File: stanford_sites_helper.install [hunk 3]
```diff
@@ -8,6 +8,29 @@
...
+        'title' => $t('File Paths'),
+        'value' => $t('Your public file path is %file_public_path. Your private file path is %file_private_path.', array('%file_public_path' => $file_public_path, '%file_private_path' => $file_private_path)),
+        'severity' => REQUIREMENT_INFO,
+      );
+
+      break;
+  }
+  return $requirements;
+
+}
```
Comment (sherakama) [inline stanford_sites_helper.install:26]: missing empty line after function.

File: stanford_sites_helper.install (hunk 2 above)
Comment (jbickar) [reply stanford_sites_helper.install]: I stole this from code you wrote. Blame Canada.

File: stanford_sites_helper.install (hunk 3 above)
Comment (jbickar) [reply stanford_sites_helper.install:26]: CodeClimate didn't catch that?

File: stanford_sites_helper.install (hunk 2 above)
Comment (sherakama) [reply stanford_sites_helper.install]: With the whole #metoo movement thing going on, it does appear that one can be accountable for past mistakes even if they attempt to right past wrongs.

**Pull request merged**